client-templates/bitbucket-server/accept.json.sample

```diff
@@ -174,6 +284,17 @@
         "password": "${BITBUCKET_PASSWORD}"
       }
     },
+    {
+      "//": "used to determine the full dependency tree",
+      "method": "GET",
+      "path": "/projects/:project/repos/:repo/browse*%2F*.csproj",
```

Comment:
This is one way to allow this, but since `*.csproj` captures `long/path/to/file.csproj`, I think we can simply change `...browse*/*.csproj` to `...browse*.csproj` and capture both the encoded and not encoded cases. Up to you if you wish to redo this.

Reply:
*/* enforces a / to be found. I prefer to keep it